src/js/PhotoSphereViewer.public.js

```diff
@@ -487,3 +487,45 @@ PhotoSphereViewer.prototype.startKeyboardControl = function() {
 PhotoSphereViewer.prototype.stopKeyboardControl = function() {
   window.removeEventListener('keydown', this);
 };
+
+/**
+ * Preload a panorama image and save it into internal cache.
+ * @param {String} pano - the file path
+ * @param {Function} callback - Progress callback, will receive the percentage as argument.
+ */
+PhotoSphereViewer.prototype.preloadPano = function(pano, callback) {
```

Comment:
Can't agree on that.
In my situation, I put markers on the scene to change panoramas (like a streetview tour) using the progress value to set the marker opacity.
If you dont like this feature, i'll keep it in my fork